index.js

```diff
@@ -52,12 +71,24 @@ document.addEventListener("DOMContentLoaded", () => {
     .finally(() => {
       isLoading = false;
     });
+  // console.log(sortBtns);
 });
 
 // AUXILIARY FUNCTIONS
 function displayPostList() {
   forumData["topic_list"].topics
     .filter((post) => post["category_id"] in supportedTopicCategories)
+    // testing block - delete before submit
+    // .sort(
+    //   (prevPost, nextPost) =>
+    //     parseActivityValueForSorting(
+    //       formatDateDiff(Date.now(), prevPost.bumped_at)
+    //     ) -
+    //     parseActivityValueForSorting(
+    //       formatDateDiff(Date.now(), nextPost.bumped_at)
+    //     )
+    // )
```

Comment:
What you are trying to achieve could look just like this. No need for that helper function.

```suggestion
.sort((prev, next) =>  new Date(prev.bumped_at) - new Date(next.bumped_at))
```